### PR TITLE
Fixed a type in gauze falling off message

### PR DESCRIPTION
**Pull request by DaxDupont**
## About The Pull Request

Corrects fall away to falls way.

## Changelog
:cl: Dax Dupont
spellcheck: Fixed a type in gauze falling off message.
/:cl:

### CODE REVIEW

**Comment by ShibaInuLord** [comment]
* the gauze fallss off. i think you just made a typo in the typo fixing pr

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 1]
```diff
@@ -940,7 +940,7 @@
  * As of the Wounds 2 PR, all bleeding is now bodypart based rather than the old bleedstacks system, and 90% of standard bleeding comes from flesh wounds (the exception is embedded weapons).
  * The same way bleeding is totaled up by bodyparts, gauze now applies to all wounds on the same part. Thus, having a slash wound, a pierce wound, and a broken bone wound would have the gauze
  * applying blood staunching to the first two wounds, while also acting as a sling for the third one. Once enough blood has been absorbed or all wounds with the ACCEPTS_GAUZE flag have been cleared,
- * the gauze falls off.
+ * the gauze fallss off.
```
Comment (Mothblocks) [inline code/modules/surgery/bodyparts/_bodyparts.dm:943]: :thinking:

```suggestion
 * the gauze falls off.
```

Comment (DaxDupont) [reply code/modules/surgery/bodyparts/_bodyparts.dm:943]: I replaced all'd my bad

**Comment by Mothblocks** [comment]
Undraft your PR when it is ready.

**Pull request merged**